1080i/Includes_Items.xml

```diff
@@ -38,6 +38,77 @@
         <include condition="String.IsEqual(Window(Home).Property(CustomDialogSettingsItems),Items_Settings_MPAACountries)">Items_Settings_MPAACountries</include>
         <include condition="String.IsEqual(Window(Home).Property(CustomDialogSettingsItems),Items_Settings_VideoOSDWidgets)">Items_Settings_VideoOSDWidgets</include>
         <include condition="String.IsEqual(Window(Home).Property(CustomDialogSettingsItems),Items_Settings_VideoInfoWidgets)">Items_Settings_VideoInfoWidgets</include>
+        <include condition="String.IsEqual(Window(Home).Property(CustomDialogSettingsItems),Items_Settings_InfolineAdditionalTags)">Items_Settings_InfolineAdditionalTags</include>    
+	</include>
+	
+	<include name="Items_Settings_InfolineAdditionalTags">
+        <include content="Settings_Button" description="Source" condition="Skin.HasSetting(Info.EnablePillboxStyle)">
+            <param name="id">8001</param>
+            <param name="control">radiobutton</param>
+            <param name="dialog">true</param>
+            <onclick condition="Skin.HasSetting(Infoline.DisableAdditionalTags)">Skin.SetBool(Infoline.DisableAdditionalTags, false)</onclick>
+            <onclick>Skin.ToggleSetting(Infoline.Source)</onclick>
+            <selected>!Skin.HasSetting(Infoline.DisableAdditionalTags) + Skin.HasSetting(Infoline.Source)</selected>			
+            <label>$LOCALIZE[21870]</label>
+        </include>
+        <include content="Settings_Button" description="HDR" condition="!Skin.HasSetting(Info.EnablePillboxStyle)">
+            <param name="id">8001</param>
+            <param name="control">radiobutton</param>
+            <param name="dialog">true</param>
+            <onclick condition="Skin.HasSetting(Infoline.DisableAdditionalTags)">Skin.SetBool(Infoline.DisableAdditionalTags, false)</onclick>
```

Comment:
To keep consistency in the skin code, it's preferred to use `Skin.Reset(setting)` rather than SetBool with false flag (same for all of these).

But also kind of moot if first point is taking about removing this setting anyway.